### PR TITLE
edit API changes, Deprecations, and Relocations sections.

### PR DESCRIPTION
**Pull request by stummjr**
I haven't touched updates 2 and 3 from API changes.

### CODE REVIEW

**Comment by redapple** [comment]
ah @stummjr , sorry, I committed some changes to "New Features and Enhancements" since.
can you try rebasing?

**Comment by stummjr** [comment]
sure, @redapple .

File: docs/news.rst [hunk 1]
```diff
@@ -86,21 +84,27 @@ New Features and Enhancements
...
+  It previously supported only XPath  (:issue:`1382`).
+
+- Add ExecutionEngine.close() method (:issue:`1423`).
+
     Adds a new method as a single entry point for shutting down the engine
-    and integrates it into Crawler.crawl() for graceful error handling during the crawling process.
+    and integrates it into Crawler.crawl() for graceful error handling during
+    the crawling process.
 
     TODO: explain what this does
```
Comment (eliasdorneles) [inline docs/news.rst]: this TODO can be just removed -- explanation should be left for docstring or main docs

File: docs/news.rst [hunk 2]
```diff
@@ -86,21 +84,27 @@ New Features and Enhancements
...
-- public Crawler.create_crawler method #1528 (https://github.com/scrapy/scrapy/commit/57f87b95d4d705f8afdd8fb9f7551033a7d88ee2)
+- public Crawler.create_crawler method (:issue:`1528`).
+ 
     Note: this is a Core API change
     Note: this is CrawlerRunner.create_crawler(), not Crawler.create_crawler
```
Comment (redapple) [inline docs/news.rst]: this was a note to change the wording of the first line

File: docs/news.rst [hunk 3]
```diff
@@ -108,40 +112,40 @@ API changes
         Return a Crawler object.
```
Comment (redapple) [inline docs/news.rst]: can be removed

**Pull request merged**